onnxruntime/test/util/default_providers.cc

```diff
@@ -99,9 +99,10 @@ std::unique_ptr<IExecutionProvider> MIGraphXExecutionProviderWithOptions(const O
   return nullptr;
 }
 
-std::unique_ptr<IExecutionProvider> OpenVINOExecutionProviderWithOptions(const OrtOpenVINOProviderOptions* params) {
+std::unique_ptr<IExecutionProvider> OpenVINOExecutionProviderWithOptions(const ProviderOptions* params,
+                                                                         const SessionOptions* session_options) {
 #ifdef USE_OPENVINO
-  return OpenVINOProviderFactoryCreator::Create(params)->CreateProvider();
+  return OpenVINOProviderFactoryCreator::Create(params, session_options)->CreateProvider();
 #else
   ORT_UNUSED_PARAMETER(params);
```

Comment:
Also need to mark `session_options` as unused. Otherwise, this fails to compile (warning as error):
```
C:\vss-agent\2.320.0\_work\onnxruntime\onnxruntime\onnxruntime\test\util\default_providers.cc(103,96): error C2220: the following warning is treated as an error [D:\b\Debug\onnxruntime_test_utils.vcxproj]
C:\vss-agent\2.320.0\_work\onnxruntime\onnxruntime\onnxruntime\test\util\default_providers.cc(103,96): warning C4100: 'session_options': unreferenced formal parameter [D:\b\Debug\onnxruntime_test_utils.vcxproj]
```